modules/api_login/modules.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+require_once APP_PATH.'modules/core/handler_modules.php';
```

Comment:
Please get rid of these includes. Module loading system shouldn't be part of the individual modules' code. Furthermore, core module should always be loaded.